var/spack/repos/builtin/packages/claw/package.py

```diff
@@ -38,6 +38,10 @@ class Claw(CMakePackage):
           sha256='44a3e17bf6e972db9760fc50bc0948309ee441dab1cdb11816ba675de0138549',
           when='@:2.0.2%intel')
 
+    patch('https://github.com/claw-project/claw-compiler/commit/4d8bc7a794af3651b8b61501388fc00096b23a85.patch',
```

Comment:
Can you please add a comment to explain what the patch does? Can be as simple as:
```suggestion
    # Fix the dependency preprocessing for compilers that 
    # cannot use redirection > to save file
    patch('https://github.com/claw-project/claw-compiler/commit/4d8bc7a794af3651b8b61501388fc00096b23a85.patch',
```